src/WebJobs.Script/Binding/Http/ObjectHeadersResult.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Net.Http.Headers;
+
+namespace Microsoft.Azure.WebJobs.Script.Binding
+{
+    public class ObjectHeadersResult : ObjectResult
```

Comment:
Would be nice to come up with a better name, ideally following the pattern `xxxObjectResult`